stock_available_unreserved/models/product.py

```diff
@@ -56,16 +57,10 @@ class ProductProduct(models.Model):
         compute='_compute_qty_available_not_reserved',
     )
 
-    @api.multi
-    def _product_available_not_res_hook(self, quants):
-        """Hook used to introduce possible variations"""
-        return False
-
     @api.multi
     def _prepare_domain_available_not_reserved(self):
         domain_quant = [
             ('product_id', 'in', self.ids),
-            ('contains_unreserved', '=', True),
         ]
         domain_quant_locations = self._get_domain_locations()[0]
```

Comment:
May be the issue comes from here [0] remove the children ?

Reply:
For the record, `_get_domain_locations` generate a list of domain fragments for each product in the recordset, domain which contains the filter for the locations. But the domain is the same for each product as they have the same context, so only one on them is required.